README.md

```diff
@@ -97,6 +97,17 @@ Note that most `tx_*` helper functions expose a `.with_key(key: &str)` builder f
   * No required builder calls
   * No notable optional builder calls
 
+#### Wasm
+
+* `.build_tx_instantiate2` - Predictably instantiates a CosmWasm contract.
+  * Required builder calls:
+    * `.with_code_id(code_id: u64)` - Should be the raw code ID of the contract being instantiated
+	* `.with_salt_hex_encoded(salt: &str)` - Should be a **hex-encoded** salt for instantiation
+	* `.with_msg(msg: serde_json::Value)`
+	* `.with_label(label: &str)`
+  * Notable optional builder calls:
+    * `.with_chain_name(chain_name: impl Into<String>)` - Should be on of `"osmosis" | "neutron" | "stride"` or one of the registered chain names from `.with_chain`
```

Comment:
typo: one*. Stride doesn't have cosmwasm so we can remove it from this list.

Reply:
Fixed!

Reply:
typo is still there: on -> one*

Reply:
Fixed!